heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesScheduler.java

```diff
@@ -358,11 +418,26 @@ protected ArrayNode getPorts(ObjectMapper mapper) {
     return ports;
   }
 
+  /**
+   * getFetchCommand
```

Comment:
Here and elsewhere can we not echo the method name in the javadocs, since it's redundant.